app/assets/javascripts/reducers/article_finder.js

```diff
@@ -20,6 +20,7 @@ export default function articleFinder(state = initialState, action) {
         newStateArticles[data.title] = {
           pageid: data.pageid,
           ns: data.ns,
+          fetchState: 1
```

Comment:
Don't do this with magic numbers. You can add all the states to a constants file and use the named constants instead. Otherwise, every person who reads any of the fetchState-related code will need to refer to each of these places in the reducer to understand what the fetchState values are supposed to mean.